### PR TITLE
Add e2e test for prediction buttons in triggered state

### PR DESCRIPTION
**Pull request by Piotrk39**
## Describe your changes

Adds e2e test case for prediction buttons to be enabled in triggered state.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review

### CODE REVIEW

File: tests/e2e/tests/Map/DefaultLegendLayersInTriggeredFlood.spec.ts
Comment (jannisvisser) [inline tests/e2e/tests/Map/DefaultLegendLayersInTriggeredFlood.spec.ts:1]: this is again the same file I see. can you isolate your PRs more? Unless they are intentionally building upon each other, in which case you can make sub-PRs maybe.

File: tests/e2e/tests/ChatSection/ChatPredictionButtonActive.spec.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    TriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+
+test(
+  qase(44, '[Trigger] Show prediction button is clickable'),
```
Comment (jannisvisser) [inline tests/e2e/tests/ChatSection/ChatPredictionButtonActive.spec.ts:35]: there's no written out test case in Qase yet.

File: tests/e2e/Pages/ChatComponent.ts [hunk 2]
```diff
@@ -80,12 +80,41 @@ class ChatComponent extends DashboardPage {
...
+    // Locators based on data-testid and filtered by formatted strings
+    const welcomeChatDialogue = this.chatDialogue.filter({
+      hasText: chatDialogueContent,
+    });
+
+    // Assertions
+    await expect(welcomeChatDialogue).toBeVisible();
+  }
+
   async allChatButtonsArePresent() {
```
Comment (jannisvisser) [inline tests/e2e/Pages/ChatComponent.ts:118]: I think this method name is no longer correct now right, given that we are now talking about showPrediction button also in the chat component.

File: tests/e2e/Pages/ChatComponent.ts [hunk 3]
```diff
@@ -153,6 +182,17 @@ class ChatComponent extends DashboardPage {
     await expect(newPage).toHaveURL(url);
     await newPage.close();
   }
+
+  async chatPredictionButtonsAreActive() {
```
Comment (jannisvisser) [inline tests/e2e/Pages/ChatComponent.ts:186]: maybe rename to 'showPredictionButtonsAreActive'? It's already in ChatComponent.ts, so no need to repeat chat maybe.

**Review by jannisvisser — CHANGES_REQUESTED** [review]

File: tests/e2e/Pages/ChatComponent.ts (hunk 3 above)
Comment (Piotrk39) [reply tests/e2e/Pages/ChatComponent.ts:186]: Changed to: 'predictionButtonsAreActive'

File: tests/e2e/Pages/ChatComponent.ts (hunk 2 above)
Comment (Piotrk39) [reply tests/e2e/Pages/ChatComponent.ts:118]: Changed to: allButtonsArePresent

**Pull request closed without merging**